app/pages/WhatIsThePropertyOrLandAddressPage.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import models.UKAddress
+import mapping.reads.Assets
+import play.api.libs.json.JsPath
+
+
+final case class WhatIsThePropertyOrLandAddressPage(index: Int) extends QuestionPage[UKAddress] {
```

Comment:
Should this throughout be named as `WhatIsThePropertyOrLandUKAddressPage` ?

Reply:
complete